### PR TITLE
port painkillers and antidepressants

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

Ported from Cosmatic Drift. Adds 3 painkillers and 6 antidepressants, as well as a chemical for managing addictions. They give you RP prompts that you might or might not follow. They shouldn't be too intrusive. THC, Space Drugs, Desoxyephedrine and Impedrezene were also made addictive.

## Why / Balance

These reagents are primarily for RP and do not provide any mechanical advantages. With the Submarine station and the psych ward, I think it has great potential for medical RP. There is a psychologist job already, might as well give them something to prescribe.

## Media

obraz_2024-05-23_170647855
obraz_2024-05-23_170707424
obraz_2024-05-23_170711771
obraz_2024-05-23_170715949
obraz_2024-05-23_170719954
obraz_2024-05-23_170725573
obraz_2024-05-23_170742229
obraz_2024-05-23_170745825
obraz_2024-05-23_170749405
obraz_2024-05-23_170753424

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added new anxiety meds, antidepressants and painkillers. Psychologists rejoice!
- add: THC, Space Drugs, Desoxyephedrine and Impedrezene are now addictive.

### CODE REVIEW

**Comment by Bonktrauma** [comment]
YES, PLEASE. I LOVE PSYCH RP

**Comment by noctyrnal** [comment]
Holy shit fuck yeah

File: Resources/Prototypes/DeltaV/Recipes/Reactions/medicine.yml
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Recipes/Reactions/medicine.yml:1]: up to null but these should be in _CD folders, because these are from cosmatic drift

Comment (MilonPL) [reply Resources/Prototypes/DeltaV/Recipes/Reactions/medicine.yml:1]: first time contributing, should be fine now I hope

**Comment by IamVelcroboy** [comment]
Maybe beyond scope of this PR but if these get approved, it would be nice to have some pre-filled pills/containers for mapping. Could potentially add to the random pill container as well.

**Review by A-z-z-y — APPROVED** [review]
Looks great, I love the role-play aspects PRs like this provide :)

(also, psych is great)

**Comment by dffdff2423** [comment]
Oh god, I just discovered this exists. Most of these are from CD's early days when we had much more lax coding standards. The addiction system is stupid and has been something I have wanted to overhaul for a half a year at this point. Many of the chems are not balanced around a environment where the playerbase cannot be trusted to not use them for antagonistic activity.

**Comment by MilonPL** [comment]
> Oh god, I just discovered this exists. Most of these are from CD's early days when we had much more lax coding standards. The addiction system is stupid and has been something I have wanted to overhaul for a half a year at this point. Many of the chems are not balanced around a environment where the playerbase cannot be trusted to not use them for antagonistic activity.

Yeah the players have figured out how to mix it into a deadly poison already. I've been planning on reworking it into StatusEffects for a while now.

**Comment by Avalon-Proto** [comment]
> Oh god, I just discovered this exists. Most of these are from CD's early days when we had much more lax coding standards. The addiction system is stupid and has been something I have wanted to overhaul for a half a year at this point. Many of the chems are not balanced around a environment where the playerbase cannot be trusted to not use them for antagonistic activity.

Its Honestly kinda funny, it took people like a day after this was merged to figure out how to make some absolutely evil poisons out of these chems.

**Comment by deltanedas** [comment]
i love my psych med 1 shot stuns and kills

**Pull request merged**